packages/main/src/Input.js

```diff
@@ -1071,6 +1095,7 @@ class Input extends UI5Element {
 
 		this.isTyping = false;
 		this.openOnMobile = false;
+		this._forceOpen = false;
```

Comment:
Is this necessary? It seems that when the suggestion is selected the popover closes and the flag will get reset in _afterClosePopover()

Reply:
Without it being here, the suggestions popover doesn't close when selecting an item.